<launch>

	<arg name="state_inspection" default="2" />


	<node name="waypoint_loader_node" pkg="waypoint_maker" type="waypoint_loader_node" output="screen">
		<param name="state_inspection" value="$(arg state_inspection)" />
	
	</node>
  <node type="rviz" name="rviz" pkg="rviz" args="-d $(find waypoint_maker)/rviz/path.rviz"/>

</launch>
